manager/templates/default/element/tv/renders/inputproperties/date.tpl

```diff
@@ -58,111 +60,145 @@ MODx.load({
-        ,fieldLabel: _('latest_date')
-        ,description: MODx.expandHelp ? '' : _('latest_date_desc')
-        ,name: 'inopt_maxDateValue'
-        ,id: 'inopt_maxDateValue{/literal}{$tv|default}{literal}'
-        ,value: params['maxDateValue'] || ''
-        ,width: 200
-        ,listeners: oc
-        ,format: MODx.config.manager_date_format
-    },{
-        xtype: MODx.expandHelp ? 'label' : 'hidden'
-        ,forId: 'inopt_maxDateValue{/literal}{$tv|default}{literal}'
-        ,html: _('latest_date_desc')
-        ,cls: 'desc-under'
+        layout: 'column'
+        ,border: false
+        ,defaults: {
+            layout: 'form'
+            ,labelAlign: 'top'
+            ,labelSeparator: ''
+            ,anchor: '100%'
+            ,border: false
+        }
+        ,items: [{
+            columnWidth: .5
+            ,items: [{
+                xtype: 'datefield'
+                ,fieldLabel: _('earliest_date')
+                ,description: MODx.expandHelp ? '' : _('earliest_date_desc')
+                ,name: 'inopt_minDateValue'
+                ,id: 'inopt_minDateValue{/literal}{$tv|default}{literal}'
+                ,value: params['minDateValue'] || ''
+                ,anchor: '98%'
+                ,listeners: oc
+                ,format: MODx.config.manager_date_format
+            },{
+                xtype: MODx.expandHelp ? 'label' : 'hidden'
+                ,forId: 'inopt_minDateValue{/literal}{$tv|default}{literal}'
+                ,html: _('earliest_date_desc')
+                ,cls: 'desc-under'
+            }]
+        },{
+            columnWidth: .5
+            ,items: [{
+                xtype: 'datefield'
+                ,fieldLabel: _('latest_date')
+                ,description: MODx.expandHelp ? '' : _('latest_date_desc')
+                ,name: 'inopt_maxDateValue'
+                ,id: 'inopt_maxDateValue{/literal}{$tv|default}{literal}'
+                ,value: params['maxDateValue'] || ''
+                ,anchor: '98%'
+                ,listeners: oc
+                ,format: MODx.config.manager_date_format
+            },{
+                xtype: MODx.expandHelp ? 'label' : 'hidden'
+                ,forId: 'inopt_maxDateValue{/literal}{$tv|default}{literal}'
+                ,html: _('latest_date_desc')
+                ,cls: 'desc-under'
+            }]
+        }]
     },{
-        xtype: 'timefield'
-        ,fieldLabel: _('latest_time')
-        ,description: MODx.expandHelp ? '' : _('latest_time_desc')
-        ,name: 'inopt_maxTimeValue'
-        ,id: 'inopt_maxTimeValue{/literal}{$tv|default}{literal}'
-        ,value: params['maxTimeValue'] || ''
-        ,width: 200
+        xtype: 'textfield'
+        ,fieldLabel: _('disabled_dates')
+        ,description: MODx.expandHelp ? '' : _('disabled_dates_desc')
+        ,name: 'inopt_disabledDates'
+        ,id: 'inopt_disabledDates{/literal}{$tv|default}{literal}'
+        ,value: params['disabledDates'] || ''
+        ,anchor: '99%'
```

Comment:
Does `anchor: '100%'` not work?

Reply:
Same for the other 99%

Reply:
Unfortunately, no, it doesn’t look very good, the fields go beyond the screen.
Although, for example, in https://github.com/modxcms/revolution/pull/14731 there is no such problem. I do not understand why.

Reply:
@Mark-H @JoshuaLuckers maybe you have some thoughts about why this is happening (I'm talking about `anchor:'100%'`) and how to fix it? :)

Reply:
When the anchor is set to 100% the div with class `x-panel-bwrap` (child from div with ID `modx-panel-tv-input-properties`) will overflow horizontally.

Reply:
Adding the `container` class on line 24 improves it a bit but still doesn't look perfect. Maybe this helps you find a better solution?

Reply:
Maybe there is a border on a 100% width somewhere causing this issue.

Reply:
The simplest solution, it seems to me, is to put a lower %.
Neither extra classes `,cls: '...'`, nor styles `,bodyStyle: '...'`, nor a border `,border: false` - something all the same go beyond.